dart/tool/dart/generate_dart.dart

```diff
@@ -1388,8 +1395,14 @@ class Type extends Member {
             gen.writeln("${field.generatableName} = "
                 "new List<${fieldType.listTypeArg}>.from($ref);");
           } else {
-            gen.writeln("${field.generatableName} = "
-                "new List<${fieldType.listTypeArg}>.from(createServiceObject($ref));");
+            // Special case `InstanceSet`.
```

Comment:
Perhaps expand on this comment a bit: 'pre version xyz, InstanceSet's values were sent in a samples field; after that, in an instances field'.

Reply:
done.